boards/nordic/nrf54l15dk/Kconfig

```diff
@@ -0,0 +1,34 @@
+# nRF54L15 DK board configuration
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF54L15DK_NRF54L15_CPUAPP_NS
+
+config NRF_MPC_REGION_SIZE
+	hex
+	default 0x1000
+	help
+	  Region size for the Memory Protection Controller (MPC) in bytes.
```

Comment:
move to dts (see nrf7x in ncs)

Reply:
nrf7 doesn't support TF-M yet. I will have to check what sideffects this will give. I would prefer to keep the scope of this PR as is since its is already big and create a task to move to DTS later. Then the change can happen in NCS and in upstream Zephyr at the same time.

Reply:
this is a hardware property, it needs to move to dts, it's value can still be accessed by Kconfig using the devicetree functions in Kconfig. Move this binding upstream https://github.com/nrfconnect/sdk-nrf/blob/main/dts/bindings/mpc/nordic%2Cnrf-mpc.yaml then add it to dts with https://github.com/nrfconnect/sdk-nrf/blob/main/dts/common/nordic/nrf7120_enga.dtsi#L80

Reply:
I added the nrf-mpc dts entry. Have a look again.